### DISCUSSION TITLE
Newly deployed project results in a 404

### DISCUSSION THREAD

**Discussion by JamesSingleton** [vercel/community#3331] [Help] [answered]
### Summary

I recently created a Next.js project and deployed it via Vercel. However, the URL results in a 404 even though everything built. Granted I will eventually change this URL, but while I am developing it, it would be nice to not have a 404. When running `npm run build && npm start` locally everything loads as expected, I mean it's just a `page.tsx` that says "Welcome" at the moment.

### Example

https://github.com/JamesSingleton/forever-singletons

### Steps to Reproduce

Go to the link in the repo and observe the 404?

**Comment by swarnava** [comment, marked as answer]
Hi,

Could you set `Framework Preset` to `Next.js` from here and rebuild: https://vercel.com/jamesrsingleton/forever-singletons/settings?

**Reply by JamesSingleton** [reply]
Oh that is weird... I wonder if there was a race condition from when I pushed to GitHub and when I imported the project into Vercel but looks like everything is working now.
